### DISCUSSION TITLE
求解，doris自建的表的key的值是区分大小写的么？能修改么？

### DISCUSSION THREAD

**Discussion by luciferhs** [apache/doris#27042] [A - General / Q&A]
其中一条数据
ord_status FILLED
account_id 1234455-CED-USDT
通过查询语句如下都是大写才能查询到，变成小写是查询不到的，如何能不区分大小写
select * from ods_ex_future_order where ord_status='FILLED'   AND account_id  like ('%USDT%') order by modified desc limit 1\G;

**Comment by zclllyybb** [comment]
两边都用upper套一下
